services/movie.py

```diff
@@ -0,0 +1,37 @@
+from typing import Optional
+
+from db.models import Movie
+
+
+def get_movies(
+        genres_ids: Optional[list] = None,
+        actors_ids: Optional[list] = None
```

Comment:
add additional typehint for list inner objects like list[YOUR_VALUE_TYPE]
for example: list that contain string elements has typehint list[str]